Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.cpp

```diff
@@ -435,12 +437,6 @@ void CDMInstanceOpenCDM::removeSessionData(const String& sessionId, LicenseType,
         keys.append(std::pair<Ref<SharedBuffer>, MediaKeyStatus> {*initData, keyStatus});
         callback(WTFMove(keys), std::nullopt, SuccessValue::Failed);
     }
-
-    SharedBuffer* initData = sessionIdMap.get(sessionId);
-    MediaKeyStatus keyStatus = getKeyStatus(responseMessage);
-    keys.append(std::pair<Ref<SharedBuffer>, MediaKeyStatus>{*initData, keyStatus});
```

Comment:
I saw this code a while ago and noticed with was very wrong. In this case we're supposed to be returning key statuses, not init data! But it's not clear from your patch series why you're removing it here, and what fixes it. I guess the changes to the MediaKeySession, but comments please! Also, this does not belong inside a commit that's moving classes in the platform layer. I've already commented it, but you need to rework your patches into logical steps.